### ISSUE TITLE
License file not distributed in package

### ISSUE THREAD

**Issue by scopatz** [asciimoo/exrex#37]
If you download the source tarball from PyPI, the license is not present

**Comment by adisakshya** [comment]
Actually if you see the MANIFEST.in file it includes "COPYING" and "README.markdown"
here "COPYING" is the License file that is in the root directory.

Hope it helps.

**Comment by adisakshya** [comment]
> If you download the source tarball from PyPI, the license is not present

Actually if you see the MANIFEST.in file it includes "COPYING" and "README.markdown" here "COPYING" is the License file that is in the root directory. Hope it helps.

**Comment by scopatz** [comment]
I don't think that is working properly then.  If you go to [here](https://files.pythonhosted.org/packages/68/5a/1f0940e6bc490825bab1d3e9441cd90f6a27d1c6022db1b12772f38de04b/exrex-0.10.5.tar.gz), and download the package, and then unzip it:

```bash
scopatz@artemis ~/downloads/t $ tar xvf ../exrex-0.10.5.tar.gz
exrex-0.10.5/
exrex-0.10.5/PKG-INFO
exrex-0.10.5/setup.py
exrex-0.10.5/exrex.egg-info/
exrex-0.10.5/exrex.egg-info/PKG-INFO
exrex-0.10.5/exrex.egg-info/entry_points.txt
exrex-0.10.5/exrex.egg-info/top_level.txt
exrex-0.10.5/exrex.egg-info/SOURCES.txt
exrex-0.10.5/exrex.egg-info/dependency_links.txt
exrex-0.10.5/exrex.py
exrex-0.10.5/setup.cfg
```

you can see that the COPYING file is not present

**Comment by adisakshya** [comment]
Yes this is not working properly.
I have improved setup.py file in PULL_REQUEST_#39, and
I have tested modules on TestPyPi and there everything is working fine
Use this link to download source tarball(SHA256): [here](https://test-files.pythonhosted.org/packages/fc/63/7b263c5b9d42db3f3718302c192b21f5adc87b7f9d591fa929c78f673677/exrex-0.10.14.tar.gz)

```
exrex-0.10.13/
exrex-0.10.13/COPYING
exrex-0.10.13/exrex.py
exrex-0.10.13/MANIFEST.in
exrex-0.10.13/PKG-INFO
exrex-0.10.13/README.md
exrex-0.10.13/setup.cfg
exrex-0.10.13/setup.py
exrex-0.10.13/exrex.egg-info/
exrex-0.10.13/exrex.egg-info/PKG-INFO
exrex-0.10.13/exrex.egg-info/entry_points.txt
exrex-0.10.13/exrex.egg-info/top_level.txt
exrex-0.10.13/exrex.egg-info/SOURCES.txt
exrex-0.10.13/exrex.egg-info/dependency_links.txt
```

Now you can see that COPYING is included.

**Comment by scopatz** [comment]
Yes, I can confirm that it is fixed in testing.

**Comment by adisakshya** [comment]
Yeah ! Fix will be live once pull_request is merged.
